0009/README.md

```diff
@@ -322,3 +322,41 @@ The NATS message payload is an Avro object with the following schema ([0009-ep-u
     ]
 }
 ```
+
```

Comment:
Traffic volume reporting is not a part of lifecycle events. Create a separate section, e.g. "Traffic reporting".